file-formats/cdbo/cdbo/type/zif_aff_cdbo_v1.intf.abap

```diff
@@ -0,0 +1,95 @@
+INTERFACE zif_aff_cdbo_v1
+  PUBLIC.
+  "! <p class="shorttext">Object Type</p>
+  "! Object type
+  "! $values {@link zif_aff_cdbo_v1.data:co_object_type}
+  TYPES ty_object_type TYPE c LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">Object Type</p>
+    "! Object type
+    BEGIN OF co_object_type,
+      "! <p class="shorttext">Tables</p>
+      "! Tables
+      tables    TYPE ty_object_type VALUE '01',
+      "! <p class="shorttext">CDS Views</p>
+      "! CDS views
+      cds_views TYPE ty_object_type VALUE '02',
+    END OF co_object_type.
+
+  "! <p class="shorttext">Field Nature</p>
+  "! Nature of the field
+  "! $values {@link zif_aff_cdbo_v1.data:co_field_nature}
+  TYPES ty_field_nature TYPE c LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">Field Nature</p>
+    "! Nature of the field
+    BEGIN OF co_field_nature,
+      "! <p class="shorttext">General fields that do not contain sensitive personal info.</p>
+      "! General fields that do not contain sensitive personal info.
+      general TYPE ty_field_nature VALUE 'GE',
+      "! <p class="shorttext">Sensitive personal information fields including key fields</p>
+      "! Sensitive personal information fields including key fields
+      sensitive    TYPE ty_field_nature VALUE 'SP',
+      "! <p class="shorttext">Non Business Information</p>
+      "! Non business information
+      non_business_information TYPE ty_field_nature VALUE 'NB',
```

Comment:
We should definitely remove the `.` at the end of titles and descriptions. The title shall be title case.

Additionally, just an idea/question: I am wondering whether it make sense to have shorter titles for the field nature values. I add a suggestion just as an example: 

```suggestion
      "! <p class="shorttext">General Information (No Sensitive Personal Information)</p>
      "! General fields that do not contain sensitive personal information
      general TYPE ty_field_nature VALUE 'GE',
      "! <p class="shorttext">Sensitive Personal Information</p>
      "! Sensitive personal information fields including key fields
      sensitive    TYPE ty_field_nature VALUE 'SP',
      "! <p class="shorttext">Non Business Information</p>
      "! Non business information
      non_business_information TYPE ty_field_nature VALUE 'NB',
```

For the field `general`, I am not sure whether it is fine to even skip "(No Sensitive Personal Information)".